src/Quantum.Development.Kit/Microsoft.Quantum.Development.Kit.nuspec

```diff
@@ -20,7 +20,7 @@
     <tags>Quantum Q# QSharp</tags>
 
     <dependencies>
-      <dependency id="Microsoft.Quantum.Simulators" version="$version$" />
+      <dependency id="Microsoft.Quantum.Simulators" version="0.16.2105143425-alpha" />
```

Comment:
Just checking if the removal of `$version$` is intentional.

Reply:
In general, it is better if it uses $version$ so they are all in-sync. In practice, this get updated with updateQDKVersion (and this package is deprecated), so I'm going keep the specific version in it.